### DISCUSSION TITLE
dp test only select a very small fraction of the test data

### DISCUSSION THREAD

**Discussion by lnnbig** [deepmodeling/deepmd-kit#2063] [Q&A]
I use "do test" option to test my data which contains 10000 configurations with the command like: dp test -m graph.pb -s /path/to/system -n 10000
However, only a very small faction of the data was selected. It seems that “dp test” randomly selected these data. How could I force the code to go testing through all data?

**Comment by njzjz** [comment]
As a designed behavior, `dp test` only tests the last set in each system.
